interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is the command language used to access repositories, make branches, etc, on GitHub
 
-  Researched answer:
+  Researched answer: Git is the version control system dealing with source code tracking, and GitHub is the hosting service for git repositories/source codes
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: And &&, Or ||, and Not !==
 
-  Researched answer:
+  Researched answer: &&, ||, !== are the primary boolean operators for JavaScript
```

Comment:
Yes, logical operators give a Boolean value, as well as relational and equality operators.